libcxx/include/__algorithm/ranges_clamp.h

```diff
@@ -37,9 +37,10 @@ struct __fn {
     _LIBCPP_ASSERT_UNCATEGORIZED(!bool(std::invoke(__comp, std::invoke(__proj, __high), std::invoke(__proj, __low))),
```

Comment:
I think we should also make a similar fix in `std::clamp` (the non-ranges version), and add a test in `libcxx/test/std/algorithms/alg.sorting/alg.clamp/clamp.comp.pass.cpp`.

Reply:
I think we don't need a similar fix in `std::clamp` because `std::clamp` doesn't support using a projection

Reply:
You're right, sorry for the noise. It supports a custom comparator, not a custom projection. I read too fast.